internal/cli/atlas/livemigrations/validation/create.go

```diff
@@ -0,0 +1,121 @@
+	sourceProjectID             string
+	sourceSSL                   bool
+	sourceManagedAuthentication bool
+	sourceUserName              string
+	sourcePassword              string
+	store                       store.LiveMigrationValidationsCreator
+}
+
+func (opts *CreateOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(store.AuthenticatedPreset(config.Default()))
+	return err
+}
+
+var createTemplate = `ID	PROJECT ID	SOURCE PROJECT ID	STATUS
+{{.ID}}	{{.GroupID}}	{{.SourceGroupID}}	{{.Status}}`
+
+func (opts *CreateOpts) Run() error {
+	createRequest := opts.newValidationCreateRequest()
+
+	r, err := opts.store.CreateValidation(opts.ConfigProjectID(), createRequest)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *CreateOpts) newValidationCreateRequest() *mongodbatlas.LiveMigration {
+	return &mongodbatlas.LiveMigration{
+		Source: &mongodbatlas.Source{
+			ClusterName:           opts.sourceClusterName,
+			GroupID:               opts.sourceProjectID,
+			Username:              opts.sourceUserName,
+			Password:              opts.sourcePassword,
+			SSL:                   pointy.Bool(opts.sourceSSL),
+			CACertificatePath:     opts.sourceCACertificatePath,
+			ManagedAuthentication: pointy.Bool(opts.sourceManagedAuthentication),
+		},
+		Destination: &mongodbatlas.Destination{
+			ClusterName: opts.destinationClusterName,
+			GroupID:     opts.ConfigProjectID(),
+		},
+		MigrationHosts: opts.migrationHosts,
+		DropEnabled:    pointy.Bool(opts.destinationDropEnabled),
+	}
+}
+
+// mongocli atlas liveMigrations|lm validation create --clusterName clusterName --migrationHosts hosts --sourceClusterName clusterName --sourceProjectId projectId [--sourceSSL] [--sourceCACertificatePath path] [--sourceManagedAuthentication] [--sourceUserName userName] [--sourcePassword password] [--drop] [--projectId projectId].
+func CreateBuilder() *cobra.Command {
+	opts := &CreateOpts{}
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create one new validation request.",
+		Long:  "Your API Key must have the Organization Owner role to successfully run this command.",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(
+				opts.ValidateOrgID,
+				opts.initStore,
+				opts.InitOutput(cmd.OutOrStdout(), createTemplate),
+			)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Run()
+		},
+	}
+	cmd.Flags().StringVar(&opts.OrgID, flag.ProjectID, "", usage.ProjectID)
+	cmd.Flags().StringVar(&opts.sourceClusterName, flag.LiveMigrationSourceClusterName, "", usage.LiveMigrationSourceClusterName)
+	cmd.Flags().StringVar(&opts.sourceProjectID, flag.LiveMigrationSourceProjectID, "", usage.LiveMigrationSourceProjectID)
+	cmd.Flags().StringVar(&opts.sourceUserName, flag.LiveMigrationSourceUsername, "", usage.LiveMigrationSourceUsername)
+	cmd.Flags().StringVar(&opts.sourcePassword, flag.LiveMigrationSourcePassword, "", usage.LiveMigrationSourcePassword)
```

Comment:
[nit] when dealing with username password we have allowed the use of short options `-p` and `-u`, also we have made password optionals to be read by stdin for security reasons, check https://github.com/mongodb/mongocli/blob/d2ad3b240a0a69e0a45a0819967eea9400a80940/internal/cli/atlas/dbusers/create.go#L116-L124

this link to other comment on `sourceManagedAuthentication` to add some extra logic to the cli here to make the flow a bit more guided and less reliant on API errors